awslambda/src/test/scala/docs/scaladsl/Examples.scala

```diff
@@ -20,20 +17,37 @@ object Examples {
   implicit val mat = ActorMaterializer()
   //#init-mat
 
-  //#init-client
-  val credentials = AwsBasicCredentials.create("x", "x")
-  implicit val lambdaClient: LambdaAsyncClient = LambdaAsyncClient
-    .builder()
-    .credentialsProvider(StaticCredentialsProvider.create(credentials))
-    .build()
-  //#init-client
-
-  //#run
-  val request = InvokeRequest
-    .builder()
-    .functionName("lambda-function-name")
-    .payload(SdkBytes.fromUtf8String("test-payload"))
-    .build()
-  Source.single(request).via(AwsLambdaFlow(1)).runWith(Sink.seq)
-  //#run
+  def initClient(): Unit = {
+    //#init-client
+    import com.github.matsluni.akkahttpspi.AkkaHttpClient
+    import software.amazon.awssdk.auth.credentials.{AwsBasicCredentials, StaticCredentialsProvider}
+    import software.amazon.awssdk.services.lambda.LambdaAsyncClient
+
+    val credentials = AwsBasicCredentials.create("x", "x")
```

Comment:
[minor] I know isn't new code, but maybe we should add a comment link to best practices for AWS key management so users don't just copy and paste this and hardcode credentials.  I found this resource: https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html